ui/src/status/fixtures.js

```diff
@@ -0,0 +1,71 @@
+export const fixtureStatusPageCells = [
```

Comment:
Yes, actually, because it's a fixture for the Status Page cells that gives them height/width as well as the query for the Bar Graph. Until we decide to move this to the server, it's stored here.